modular_skyrat/modules/customization/modules/language/xenoknockoff.dm

```diff
@@ -1,9 +0,0 @@
-/datum/language/xenoknockoff
```

Comment:
This was added so that Xenohybrid players stopped complaining over the fact they couldn't talk a language like Xenomorphs.

Reply:
Wrong PR